src/org/loklak/harvester/YoutubeScraper.java

```diff
@@ -134,28 +135,25 @@ public static JSONObject parseVideo(final BufferedReader br) throws IOException
                 if ((p = input.indexOf("yt-subscriber-count")) >= 0) {
                     String subscriber_string = parseProp(input, p, "title");
                     if (subscriber_string == null) continue;
-                    json.put("youtube_subscriber", parseNumber(subscriber_string));
+                    json.put("youtube_subscriber", parseValue(subscriber_string));
                     continue;
                 }
                 if (input.indexOf("\"like this") > 0 && (p = input.indexOf("yt-uix-button-content")) >= 0) {
                     String likes_string = parseTag(input, p);
-                    json.put("youtube_likes", parseNumber(likes_string));
+                    json.put("youtube_likes", parseValue(likes_string));
```

Comment:
Hey @vibhcool . number of likes are always in numbers format . Only subscribers are shown in M,B,K format.  You can remove this.